src/Ledger/Gov.lagda

```diff
@@ -110,13 +110,13 @@ validHFAction _ _ _ = ⊤
 \footnotetext{\AgdaBound{l}~\AgdaFunction{∷ʳ}~\AgdaBound{x} appends element \AgdaBound{x} to list \AgdaBound{l}.}
 \begin{figure*}[h]
 \begin{AgdaMultiCode}
-\begin{code}[hide]
--- convert list of (GovActionID,GovActionState)-pairs to list GovActionID pairs.
+\begin{code}
```

Comment:
This isn't what we wanted, this stuff was intentionally hidden (you've even proposed that). So the figure should be fixed without showing that again.

Reply:
Yes, I even proposed that, but in response to that proposal you said we would have to add an explanation to the prose if we hide this code, which I thought was a worse option than simply showing the code with a comment.

Reply:
I've gone ahead and removed the functions and added a paragraph describing the figure and mentioning the hidden function and their purpose.